docs/source/DeveloperGuide/AddingNewComponents/Utilities/gLogger/Changes/index.rst

```diff
@@ -279,6 +279,93 @@ will stay the same:
     # > message
     # stderr
     # > VERBOSE: message
+    
+*Backend* configuration
+----------------------
+
+This is an important change which requires some modifications of your code if you want
```

Comment:
Here we do not document the changes. here we document what it is at a point in time. The changes, the transitions, should be described in the wiki of the repository.